devbuild.sh

```diff
@@ -68,9 +68,22 @@ fi
 # Source the README file for this platform/compiler combination, then build the code
 . $ENV_FILE
 
+cp src/CMakeLists.txt_model src/CMakeLists.txt
 mkdir -p ${BUILD_DIR}
 cd ${BUILD_DIR}
-cmake .. -DCMAKE_INSTALL_PREFIX=..
+cmake .. -DCMAKE_INSTALL_PREFIX=.
 make -j ${BUILD_JOBS:-4}
+mv bin ../.
+
+cd ..
+cp src/CMakeLists.txt_DA src/CMakeLists.txt
+. ${ENV_FILE}_DA
+rm -fr ${BUILD_DIR}
+mkdir -p ${BUILD_DIR}
+cd ${BUILD_DIR}
+cmake .. -DCMAKE_INSTALL_PREFIX=.
+make -j ${BUILD_JOBS:-4}
+cd bin
+cp * ../../bin/.
```

Comment:
I don't think this is the appropriate approach for using CMake. If we do need to separate the GSI build from the rest of the build, then I'd propose that you do that in a way that keeps the current components built as-is and do something "special" only for the GSI portion. This also completely bypasses the CMakeLists.txt file in the top-level SRW App directory.

Reply:
See my answer to Jeff's comment. And answer below you big question.